lookout/style/typos/benchmarks/typo_commits_report.py

```diff
@@ -42,23 +44,33 @@ def run(self, ptr: ReferencePointer, data_service: DataService) -> Iterable[Typo
         """
         for file in request_files(data_service.get_data(), ptr, contents=True, uast=True,
                                   unicode=False):
-            if file.path == self.config["filepath_to_analyze"]:
+            if file.path == self.config["analyze"]["filepath"]:
                 break
         else:
-            raise ValueError("No such file %s in %s" % (self.config["filepath_to_analyze"], ptr))
-
-        typos_fixes = list(self.generate_typos_fixes([
-            UnicodeChange(head=file, base=File(path=file.path, language=file.language))]))
+            raise ValueError("No such file %s in %s" % (self.config["analyze"]["filepath"], ptr))
+        line = self.config["analyze"]["line"] + 1
```

Comment:
It looks like we have 0-based lines numbers in the dataset.

Reply:
Yes, most probably